students/ahyounglee/users/views.py

```diff
@@ -41,4 +41,26 @@ def post(self, request):
             return JsonResponse({"message": "KEY_ERROR"}, status=400)
 
         except ValidationError as e:
-            return JsonResponse({"message": e.message}, status=400)
\ No newline at end of file
+            return JsonResponse({"message": e.message}, status=400)
+
+class LoginView(View):
+    def post(self, request):
+        data = json.loads(request.body)
+        
+        try:
+            email    = data['email']
+            password = data['password']
+
+            if not User.objects.filter(email=email).exists():
+                raise ValidationError('INVALID_USER')
```

Comment:
email하고 password가 동시에 만족하는 유저를 찾아야 하므로 다음과 같이 수정하면 좋을 것 같습니다.
```suggestion
            if not User.objects.filter(email=email, password=password).exists():
                raise ValidationError('INVALID_USER')
```

Reply:
수정완료했습니다!